src/codec_svt.c

```diff
@@ -9,6 +9,9 @@
 
 #include <stdint.h>
 #include <string.h>
+#if SVT_AV1_CHECK_VERSION(3, 0, 0)
+#include <stdbool.h>
+#endif
```

Comment:
Optional: We can include <stdbool.h> and use the constant `true` unconditionally.

Reply:
Sorry, too used to work with C89 in WebP :)